node/node_test.go

```diff
@@ -96,6 +96,20 @@ func TestSplitAndTrimEmpty(t *testing.T) {
 	}
 }
 
+func TestCompanionInitialHeightSetup(t *testing.T) {
```

Comment:
Why is this part of this PR? MY guess is its because of a merge that did not fully go through?